Editor/languages/editor_en.ts

```diff
@@ -6507,7 +6507,9 @@ The special state is defined by the active configuration pack.</translation>
         <source>Show a blank screen instead of loading screen.
 In the Legacy Engine level loading screen, it would otherwise 
 show the current playable character(s) and number of lives.</source>
-        <translation type="unfinished"></translation>
+        <translation>Show a blank screen instead of the standard loading screen.
+This will prevent the Legacy Engine from displaying the player&apos;s
```

Comment:
This is not for legacy engine, this feature introduced in SMBX-38A. the description does describes what it's the pre-level intro scene with showing the legacy engine as example.